src/registrar/templates/django/admin/domain_change_form.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'admin/change_form.html' %}
+
+{% block field_sets %}
+    <div class="submit-row">
+        <input type="submit" value="Place hold" name="_place_client_hold">
```

Comment:
**question (non-blocking)**: This is a huge can of worms because of how styling is done on the Django admin, but I initially expected this to have the `usa-button` class. Does it look right if we start using USWDS components inside of admin?

Reply:
I'll defer this to the new team. 😄